lit_gpt/data/alpaca_gpt4.py

```diff
@@ -0,0 +1,21 @@
+# Copyright Lightning AI. Licensed under the Apache License 2.0, see LICENSE file.
+
+
+from dataclasses import dataclass, field
+from pathlib import Path
+from lit_gpt.data.alpaca import Alpaca
+
+
+_URL = "https://raw.githubusercontent.com/Instruction-Tuning-with-GPT-4/GPT-4-LLM/main/data/alpaca_gpt4_data.json"
+
+
+@dataclass
+class AlpacaGPT4(Alpaca):
```

Comment:
Would you like to keep the `test_split_fraction` from Alpaca or define a new one? Because the size is slightly different

Reply:
Good idea. I adjusted it so that it is also 2000 test examples similar to the regular alpaca dataset